homeassistant/const.py

```diff
@@ -654,6 +654,8 @@ class UnitOfPower(StrEnum):
     """Power units."""
 
     WATT = "W"
+    WATT_HOUR = "Wh"
```

Comment:
This is not a unit of Power, it's a Unit of Energy, which should already exists. I don't think you're allowed to edit this file.

Reply:
I just pushed a commit that fixes this.